composer.json

```diff
@@ -19,11 +19,11 @@
   ],
   "require": {
     "php": "^8.0",
-    "symfony/config": "^5.4",
-    "symfony/dependency-injection": "^5.4",
-    "symfony/framework-bundle": "^5.4",
-    "symfony/http-kernel": "^5.4",
-    "symfony/property-access": "^5.4"
+    "symfony/config": "^6.2",
+    "symfony/dependency-injection": "^6.2",
+    "symfony/framework-bundle": "^6.2",
+    "symfony/http-kernel": "^6.2",
+    "symfony/property-access": "^6.2"
```

Comment:
We should support both Symfony `^5.4` and `^6.2`, otherwise it will no longer be compatible with Pimcore 10.  

That being said, Symfony `^5.4` is [the current LTS](https://symfony.com/releases/5.4), which is supported until the end of 2025, so we should do that too, unless we have a good reason not to.